HHS/Head-Start-TTADP
# [TTAHUB-753] - Text change to other entity participant

## Description of change
Changing the Other Entity participant option "Local/State Agency(ies)" to "State or Local Agency" per Adam Roux.


## How to test
In Activity Report summary section, select Other Entity under "Who is the activity for?" field, then go to the bottom of the page and select the "Other entity participants" dropdown to see the updated text.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-753


## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
